daprdocs/content/en/developing-applications/building-blocks/actors/actors-timers-reminders.md

```diff
@@ -137,6 +137,10 @@ You can remove the actor reminder by calling
 DELETE http://localhost:3500/v1.0/actors/<actorType>/<actorId>/reminders/<name>
 ```
 
+If an actor reminder is triggered and the app does not return a 2** code to the runtime (e.g. because of a connection issue),
```

Comment:
```suggestion
If an actor reminder is triggered and the app does not return a 2** code to the runtime (for example, because of a connection issue),
```